src/main/java/com/blackducksoftware/integration/hub/alert/digest/DigestRemovalProcessor.java

```diff
@@ -74,7 +74,7 @@ private String createCacheKey(final NotificationEntity entity) {
     }
 
     private boolean processPolicyNotifications(final Map<String, NotificationEntity> categoryMap, final NotificationEntity entity) {
-        final String notificationType = entity.getNotificationType();
+        final String notificationType = entity.getNotificationType().name();
         if (NotificationCategoryEnum.POLICY_VIOLATION.name().equals(notificationType)) {
```

Comment:
You can compare just enums and remove all string related expressions here

Reply:
That would require modifying the entity cache stuff. We don't want to do that for now.

Reply:
Actually, I think there's an easy way to do it.